### PR TITLE
[INS] Update Instance creation with routed IP feature

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: compute/gpu/how-to/create-manage-gpu-instance.mdx [hunk 1]
```diff
@@ -7,13 +7,18 @@ content:
...
   posted: 2022-03-25
 categories:
   - compute
 ---
 
-Scaleway GPU Instances are virtual compute instances equipped with dedicated high-end Nvidia graphical processing unit (GPUs). They are ideal for data processing, artificial intelligence, rendering and video encoding. After you have created your GPU Instance, you can [connect to it via SSH](/compute/instances/how-to/connect-to-instance) and run one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker) to access a [preinstalled environment](/compute/gpu/how-to/use-preinstalled-env) with all your [favorite AI libraries and tools](/compute/gpu/reference-content/docker-images/) already installed. In addition to this, GPU Instances have all the features of our regular Instances, including [flexible IPs](/compute/instances/how-to/use-flexips/), [Security Groups](/compute/instances/how-to/use-security-groups/), [Private Networks](/compute/instances/how-to/use-private-networks/), [backups](/compute/instances/how-to/create-a-backup/) and more. When you are done using your GPU Instance, you can easily [delete it](/compute/instances/how-to/delete-instance/) from the Scaleway console.
+Certainly!
```
Comment (jcirinosclwy) [inline compute/gpu/how-to/create-manage-gpu-instance.mdx:16]: Not sure if this was meant to be here?

Comment (bene2k1) [reply compute/gpu/how-to/create-manage-gpu-instance.mdx:16]: Certainly not

File: compute/gpu/how-to/create-manage-gpu-instance.mdx [hunk 2]
```diff
@@ -24,19 +27,24 @@ Scaleway GPU Instances are virtual compute instances equipped with dedicated hig
...
+    - **Instance Type (GPU)**: Select the desired GPU Instance type, considering factors such as processing power, memory, storage options, and bandwidth. Refer to our guide on [choosing the right GPU Instance type](/compute/gpu/reference-content/choosing-gpu-instance-type/) for more information.
+    - **Image**: Pick an operating system image suitable for your GPU Instance. For example, select **Ubuntu Jammy GPU OS 12**, which comes with preinstalled Nvidia drivers and an Nvidia Docker environment. You have the flexibility to customize your working environment using Docker with our provided Docker images or your own containers.
+    - **Volumes**: Optionally, add storage volumes for your Instance. You can adjust settings such as block and local storage volumes according to your requirements.
+        <Message type="note">
+          * The recommended minimum volume size for GPU OS images is 125 GB.
+          * If your GPU Instance support scratch storage, the scratch volume displays but can not be edited. [Learn more about scratch storage.](/compute/gpu/how-to/use-scratch-storage-h100-instances/)
```
Comment (RoRoJ) [inline compute/gpu/how-to/create-manage-gpu-instance.mdx:39]:
```suggestion
          * If your GPU Instance supports scratch storage, the scratch volume displays but can not be edited. [Learn more about scratch storage.](/compute/gpu/how-to/use-scratch-storage-h100-instances/)
```

File: compute/instances/how-to/create-an-instance.mdx [hunk 3]
```diff
@@ -27,36 +27,28 @@ Select a tab below for instructions on how to create an Instance via either our
...
-          - Select whether you want to allocate a new IPv4 to the Instance, or use one of your existing IP(s).
-          - If you want to create an Instance without a public IPv4 address, toggle the switch to disable IPv4.
-          - Tick the checkbox to assign a globally routed IPv6 address to your Instance.
-          - You can attach up to 5 IPs to an Instance, combining IPv4 and IPv6 addresses.
+        - Configure the network of the Instance. You can either select to use **Routed public IP** (a dedicated public IP address routed to your Instance that allows direct communication between the Instance and the Internet) or a **NAT public IP** (a public IP address that uses a carrier-grade NAT to translate the Instances NAT IP address). If you are unsure which to use, we recommend a routed public IP for ease of use and improved performance.
+          - Ensure the checkbox is ticked to assign a **Public IPv4** to the Instance. You can either allocate a new IPv4 address or select one or multiple existing IPv4s.
```
Comment (RoRoJ) [inline compute/instances/how-to/create-an-instance.mdx:47]:
```suggestion
          - Leave the checkbox ticked to assign a **Public IPv4** to the Instance. You can either allocate a new IPv4 address or select one or multiple existing IPv4s. Alternatively, uncheck the box if you do not want an IPv4.
```

File: compute/instances/quickstart.mdx [hunk 4]
```diff
@@ -24,42 +24,33 @@ Scaleway [Instances](/compute/instances/concepts/#instance) are computing units
...
-        - Select if you want to allocate a new IPv4 to the Instance or select one of your existing IP(s).
-        - If you want to create an Instance without public IPv4, toggle the switch to disable IPv4.
-        - Tick the checkbox to assign a globally routed IPv6 address to your Instance.
-        - You can attach up to 5 IPs to an Instance, combining IPv4 and IPv6 addresses.
+    - Configure the network of the Instance. You can either select to use **Routed public IP** (a dedicated public IP address routed to your Instance that allows direct communication between the Instance and the Internet) or a **NAT public IP** (a public IP address that uses a carrier-grade NAT to translate the Instances NAT IP address). If you are unsure which to use, we recommend a routed public IP for ease of use and improved performance.
+      - Ensure the checkbox is ticked to assign a **Public IPv4** to the Instance. You can either allocate a new IPv4 address or select one or multiple existing IPv4s.
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply compute/instances/quickstart.mdx:44]:
```suggestion
      - Leave the checkbox ticked to assign a **Public IPv4** to the Instance. You can either allocate a new IPv4 address or select one or multiple existing IPv4s. Alternatively, uncheck the box if you do not want an IPv4.
```

File: compute/instances/quickstart.mdx [hunk 5]
```diff
@@ -24,42 +24,33 @@ Scaleway [Instances](/compute/instances/concepts/#instance) are computing units
...
+      - Ensure the checkbox is ticket to assign a **Public IPv6** to the Instance. You can either allocate a new IPv6 address or select one or multiple existing IPv6s.
```
Comment (bene2k1) [inline compute/instances/quickstart.mdx:45]:
```suggestion
      - Leave the checkbox ticked to assign a **Public IPv6** to the Instance. You can either allocate a new IPv6 address or select one or multiple existing IPv46. Alternatively, uncheck the box if you do not want an IPv4.
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**